api/data_refinery_api/views.py

```diff
@@ -258,6 +264,11 @@ def perform_update(self, serializer):
                 pjda.save()
 
                 job_sent = False
+
+                obj = serializer.save()
+                if obj.email_address != supplied_email_address:
```

Comment:
Could we add a check to make sure supplied_email_address isn't empty? This would make it possible to accidentally overwrite the email address if it was set in a previous request.

Reply:
That'd also prevent you from removing an email address. I've been operating with the idea that it's okay not to supply an email if you just want to use the website and not be notified. If we're going to require an email address, we should put some bounds on that and require it everywhere.